lib/qa/authorities/getty/aat.rb

```diff
@@ -23,7 +29,7 @@ def sparql(q)
                  skos:inScheme <http://vocab.getty.edu/aat/> ;
                  gvp:prefLabelGVP [skosxl:literalForm ?name].
               FILTER regex(?name, \"#{search}\", \"i\") .
-            } LIMIT 10"
```

Comment:
Okay, we'll leave it out then. In the future it might be something we want to add back, but via a settable attribute.